PluginCore/ScintillaNet/Events.cs

```diff
@@ -19,8 +19,8 @@ namespace ScintillaNet
     public delegate void NeedShownHandler(ScintillaControl sender, int position, int length);
     public delegate void UserListSelectionHandler(ScintillaControl sender, int listType, string text);
     public delegate void URIDroppedHandler(ScintillaControl sender, string text);
-    public delegate void DwellStartHandler(ScintillaControl sender, int position);
-    public delegate void DwellEndHandler(ScintillaControl sender ,int position);
+    public delegate void DwellStartHandler(ScintillaControl sender, int position, int x, int y);
```

Comment:
After some long time I finally checked this... although I've been using this in my own build since this was done and never found any problem with other plugins.

I found the following during my tests:

- Using .NET 3.5 this is fully compatible for old plugins.
- Using .NET 4+ this would be compatible for old plugins targetting .NET 4+.
- Using .NET 4+ this would be compatible for old plugins targetting .NET 3.5.
- Using .NET 3.5 for PluginCore this would be incompatible for old plugins targetting .NET 4+ as the handler would take `y` as `position`. **This wouldn't happen anyway**